networking_monitoring.prolific

```diff
@@ -97,6 +97,99 @@ L: security
 
 ---
 
+Inspect your c2c packets with tcpdump
+
+### What?
+Sometimes a user says "container to container networking is broken! AppA can't talk to AppB". After making sure that they have c2c policies set, the next thing you might do is use tcpdump to inspect the packets.
+
+Tcpdump is a CLI tool that allows you to inspect all of the traffic (not just TCP, despite the name) flowing through your container. This is a great tool to use to debug how far packets are making it, how long the packets are taking to reach their destination, and to look at where in the TCP handshake things are failing.
+
+In this story you are going to look at the packets being sent from Boots to Swiper. Then you'll watch the packets being sent in response.
+
+### How?
+**Setup**
+1. You should have two apps pushed named Boots and Swiper.
```

Comment:
Maybe "Use the two apps we pushed In the previous story: Boots and Swiper" or something along those lines so that if someone jumps to this story they aren't confused about where these two apps come from?